src/library/dao/playlistdao.cpp

```diff
@@ -1053,3 +1059,20 @@ void PlaylistDAO::sendToAutoDJ(const QList<TrackId>& trackIds, AutoDJSendLoc loc
             break;
     }
 }
+
+void PlaylistDAO::createTablePositionList(QList<int> &positionsTable,
```

Comment:
Hint: You don't need the current positions if you order the IDs by positions. QList is ordered.